rust/Cargo.toml

```diff
@@ -35,9 +35,9 @@ categories = ["database"]
 
 [workspace.dependencies]
 adbc_core = { path = "./core", version = "0.17.0" }
-arrow-array = { version = "53.4.0", default-features = false, features = [
+arrow-array = { version = "54.1.0", default-features = false, features = [
     "ffi",
 ] }
-arrow-buffer = { version = "53.1.0", default-features = false }
-arrow-schema = { version = "53.1.0", default-features = false }
-arrow-select = { version = "53.4.0", default-features = false }
```

Comment:
Can we get a release of `arrow-adbc` that uses Datafusion 44 and Arrow 53.3.0 (which are the arrow versions used by df 44)?

I can send that PR.

Reply:
This crate is still unpublished on crates.io, but the last release (16) was based on Datafusion 44 and Arrow 53:

https://github.com/apache/arrow-adbc/blob/05c33a7700b981b3ce9097200a49357f8e9499ac/rust/driver/datafusion/Cargo.toml#L31-L32

https://github.com/apache/arrow-adbc/blob/05c33a7700b981b3ce9097200a49357f8e9499ac/rust/Cargo.toml#L38-L43

Reply:
Do we need to release more frequently here to keep up with the upstream project?

Reply:
Maybe, but not necessarily for all updates; users are only blocked when they want to bump their major versions of these crates and this driver is not released for that major version yet.

Given that this is not on crates.io yet, it doesn't really matter (because users are most likely [pulling and building directly from git](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-git-repositories)).

Reply:
Ok, event if I use `53.3.0` specifically, `arrow-adbc` won't force `53.1.0` because the only difference is the minor version.

Reply:
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio